src/task/src/pullRequests/pullRequest.ts

```diff
@@ -5,6 +5,7 @@ import { injectable } from 'tsyringe'
 import CodeMetrics from '../metrics/codeMetrics'
 import RunnerInvoker from '../runners/runnerInvoker'
 import Logger from '../utilities/logger'
+import Inputs from '../metrics/inputs'
```

Comment:
I have some setting in VS Code that sorts these in a particular order based on file paths. In this case, this line should go just after `import CodeMetrics...`

Reply:
Is that setting in the `.editorconfig` file? I'm using VS Code as well.

Reply:
I don't believe it comes from the `.editorconfig` file, but from somewhere else. It might be the following extension. If you do install this and the format option in VS Code corrects the sort order for you, let me know and I'll add this as a recommend extension for the repo:

```Text
Name: ESLint
Id: dbaeumer.vscode-eslint
Description: Integrates ESLint JavaScript into VS Code.
Version: 2.4.4
Publisher: Microsoft
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint
```